### PR TITLE
feat: update black

### PR DESCRIPTION
**Pull request by c0state**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
         "click>=7",
-        "black==21.11b1",
+        "black>=22,<23",
```
Comment (c0state) [inline setup.py:30]: Since black is "official" now, changing to use a range. (assuming black is versioned accordingly)

Comment (c0state) [reply setup.py:30]: Ah ok, that's why we pinned it I guess.

```suggestion
        "black==22.1.0",
```

**Review by sloria — APPROVED** [review]

**Comment by c0state** [comment]
Were we using travis to build still? I don't see recent builds at https://app.travis-ci.com/github/4Catalyzer/fourmat/builds but I also don't see where/how build `0.10.1` got made?

**Comment by sloria** [comment]
@c0state ah yeah, we need to migrate this to GHA. for now, go ahead and manually push the tag and publish

**Comment by c0state** [comment]
Can someone here add `stephen.liu` to the pypi project?

**Comment by sloria** [comment]
i don't have privs to add you to the project since i'm not owner. @itajaja could you please make both me and @c0state owners on the pypi repo here please? https://pypi.org/manage/projects/

**Comment by sloria** [comment]
@c0state i release 0.11.0 https://pypi.org/project/fourmat/0.11.0/

**Pull request merged**